SocialFilter/sections/specific.txt

```diff
@@ -82,6 +82,7 @@ esquire.com,dotesports.com,mashable.com,pcmag.com,onefootball.com,ign.com,israel
 !
 ! NOTE: Specific rules
 !
+novabrasilfm.com.br##.post-interna__content__compartilhamento__list:has(.post-interna__content__compartilhamento__list__item__rede-social)
```

Comment:
`compartilhamento` = `sharing` in English. It seems can be used without `:has()`
or
`novabrasilfm.com.br##.post-interna__content__compartilhamento__list__item__rede-social`